aws-auth-cognito/src/main/java/com/amplifyframework/auth/cognito/actions/FetchAuthSessionCognitoActions.kt

```diff
@@ -176,7 +177,7 @@ object FetchAuthSessionCognitoActions : FetchAuthSessionActions {
                     FetchAuthSessionEvent(FetchAuthSessionEvent.EventType.FetchAwsCredentials(it, loginsMap))
                 } ?: throw Exception("Fetching identity id failed.")
             } catch (notAuthorized: aws.sdk.kotlin.services.cognitoidentity.model.NotAuthorizedException) {
-                val exception = SignedOutException(
+                val exception = NotAuthorizedException(
```

Comment:
For v1 and iOS, not authorized is returned to customer as SignedOutException. We should mention this in docs and breaking changes list.